arex-storage-config/src/main/java/com/arextest/config/utils/parse/sqlparse/SqlParseManager.java

```diff
@@ -0,0 +1,84 @@
+package com.arextest.config.utils.parse.sqlparse;
```

Comment:
It is recommended to adjust the location of the parsing tool class. arex-storage-config is the management module for tables related to the configuration module and is referenced by arex-api.